### ISSUE TITLE
Create a website for the project

### ISSUE THREAD

**Issue by saadpasta** [kautukkundan/Awesome-Profile-README-templates#77]
@kautukkundan amazing work on the repo🔥. 
Currently, it's hard to look at the readme.md, we have to navigate through all the folder to look at them we could make a website for that using Gatsby that will help fetch readme's and show on the website. We could also use that to generate thumbnail for the readme so the user can have a look at the readme beforehand 

We could further break down readme in 
Trending 
Most Liked 
and much more.

**Comment by cristianmarint** [comment]
Sounds great!!

**Comment by kautukkundan** [comment]
Hey @saadpasta That's a great idea, building a website is in pipeline since a long time, however I am a little caught up between other things. I'll do that very soon. If you want, you can also try it out and send a PR maybe?

**Comment by cristianmarint** [comment]
@kautukkundan Are you willing to consider a PR with Gohugo?

**Comment by kautukkundan** [comment]
yes definitely!
Infact I did try experimenting with Hugo. It required contributor to additionally add their links in the "sidebar". Will it be a lot of work for the contributor? Will it make it difficult to contribute? what do you think? 
If this is possible and/or there's a better way, then please suggest.

**Comment by 98lenvi** [comment]
@kautukkundan How about using `Gastby`? We will just require the users to add the following on top of the `.md` file

```
---
title: 'README template name'
date: "2020-03-20T22:40:32.169Z"
description: README template description.
---
```

**Comment by kautukkundan** [comment]
Oh that sounds doable. 
If you're free can you add this and make a PR in a new branch "static-site" or something? Else I'll check it out on sunday.

**Comment by 98lenvi** [comment]
@kautukkundan  I will do it tonight.I will not focus heavily on styling, We can collaborate & focus on styling after the PR.

**Comment by cristianmarint** [comment]
@98lenvi Sounds great! I will also suggest something along the lines of

```
---
.
.
.
category: (...)
featured: true or false (So popular profiles can be display on mainpage)
github-username(or url): (...)
---
```
Also is Disqus integration for generate interaction doitable?

**Comment by 98lenvi** [comment]
@cristianmarint Well, I figured out that we have 73 existing README already there (and many more in pending PRs) and updating all that with the comment for title, date & description is going to be difficult, So I skipped it. Disqus is very doable.

All the pages are programmatically created based on the folder & file name.

**Comment by VinayVeerappaji** [comment]
It'll be good if the website displays the raw markdown format and I'll be willing to submit a PR for it.

**Comment by 98lenvi** [comment]
@VinayVeerappaji, It's a great idea since everyone will be able to just copy-paste the README markdown from the website and use it (without ever having to visit the GitHub repository). I think you can get started with it once my PR gets merged.

**Comment by VinayVeerappaji** [comment]
Yes, you understood my point @98lenvi. So others can able to use the same by simply modifying it.

**Comment by piyushxbajaj** [comment]
Hi I loved reading this read.☺️ Wanted to know how the website building is going?

**Comment by Alan-hanmaDio** [comment]
Em seg, 27 de jul de 2020 00:53, Piyush <notifications@github.com> escreveu:

**Comment by sudnyeshtalekar** [comment]
Really great idea of building website for all these profiles.Loved reading the conversation.How's the website build going??
